web/sidebars.js

```diff
@@ -182,6 +182,16 @@ module.exports = {
         },
       ],
     },
+    {
+      type: 'category',
+      label: 'Integrations',
+      collapsed: false,
+      items: [
+        'guides/integrations/prisma',
+        'guides/integrations/auth0',
+        'guides/integrations/draftbit',
```

Comment:
- add vercel here
- make them alphabetical order

Reply:
Good spot! Fixed and pushed 👍